includes/definitions/tplink.yaml

```diff
@@ -7,7 +7,8 @@ over:
 discovery:
     -
         sysObjectID:
-            - .1.3.6.1.4.1.11863.
```

Comment:
Any reason to be more restrictive than before ? Are you sure no device are now unselected for tplink OS

Reply:
Hi,
OID of EAP is 
             .1.3.6.1.4.1.11863.3

OID for SW and CPE-R is 
            - .1.3.6.1.4.1.11863.1.
            - .1.3.6.1.4.1.11863.5.

the effort was to separate these groups due to the different display (EAP has a sensor for the number of WiFi clients, but others such as memory or CPU are not there)

Reply:
But right now, you restrict the discovery to 1.3.6.1.4.1.11863.1. and 1.3.6.1.4.1.11863.5. So 1.3.6.1.4.1.11863.3 devices are now "Generic Device". Probably not what we want to do. 
You can discover sensors for WIFI clients if present, and skip it if not. Same fo memory and cpu. No need to filter at the OS level (which can and should remain unique unless they are really different OS)

Reply:
> You can discover sensors for WIFI clients if present, and skip it if not. Same fo memory and cpu. No need to filter at the OS level (which can and should remain unique unless they are really different OS)

that is what I was looking for, but does not found any of this in examples or ready devices. This is also reason why I create fresh OS.

Is there any example in existing code? This PR can be closed without merge then...

Reply:
I don't know your devices so I can tell for sure which solution is best. I can only look at your PR and try help you find the best solution. 

For instance, the code you added in `LibreNMS/OS/TplinkEap.php` could be simply in the standard Tplink OS PHP file with a `if` statement matching the device that has wireless code. So this would cleanly handle both situations. 

For the processor and memory, the YAML may be able to discover both standard Tplink and TplinkEap and should only keep the one that has an SNMP reply, so once again it should behave correctly.